src/modules.js

```diff
@@ -435,6 +432,14 @@ function exportRuntime() {
     'abort',
   ];
 
+  // Add JS library elements such as FS, GL, ENV, etc. These are prefixed with
+  // '$ which indicates they are JS methods.
+  for (var ident in LibraryManager.library) {
+    if (ident[0] === '$') {
```

Comment:
Are you sure non of those others can be deleted too?  Perhaps assert here that `ident.substr(1)` is not already in `runtimElements`?

Reply:
For example it looks like writeStringToMemory is in this class..

Reply:
That one is kind of tricky as the strings library works differently in minimal runtime and not.

I think it's ok if we end up with duplicates in this array - no harm is caused, as it's just used to check for existence (which probably means it could be a set, but it's around 100 elements in practice...).